src/sql/platform/telemetry/common/telemetryKeys.ts

```diff
@@ -79,6 +79,11 @@ export enum TelemetryAction {
 
 export enum NbTelemetryAction {
 	RunCell = 'RunCell',
-	RunAll = 'RunNotebook'
+	RunAll = 'RunNotebook',
+	AddCell = 'AddCell',
+	ChangeKernel = 'ChangeKernel',
+	NewNotebookFromConnections = 'NewNotebookWithConnectionProfile',
+	SearchStarted = 'SearchStarted',
```

Comment:
These seem a bit too generic to warrant being in `NbTelemetryAction` - would suggest putting in `TelemetryAction` enum and then just ensure that you have the view set properly in the events to know where the search was triggered from.